relion/protocols/protocol_base.py

```diff
@@ -1067,65 +1109,65 @@ def _setBasicArgs(self, args):
                      '--o': self._getExtraPath('relion'),
                      '--oversampling': self.oversampling.get()
                      })
-    
+
         if self.IS_CLASSIFY:
             args['--tau2_fudge'] = self.regularisationParamT.get()
             args['--iter'] = self._getnumberOfIters()
 
-            if (not self.doContinue and relion.Plugin.isVersion2Active() and
-                relion.Plugin.getActiveVersion() != V2_0):
+            if not self.doContinue:
```

Comment:
the removed code looked weird, apparently subsets appeared in 2.1 ? So should we exclude 2.0?

Reply:
Not sure here...I guess this is part of your more general comment about reviewing when the isVersion2Active...when things are available for 2.x or for 2.1 onwards.